app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_likes_counter
+    post.update(LikesCounter: post.likes.count)
+  end
```

Comment:
[OPTIONAL] It is a good practice to use rails callbacks like `after_save` in scenarios like this. This way you can make your update_likes_counter private making it more secure.